### ISSUE TITLE
Bug with typedef rule for arrow function used as parameter

### ISSUE THREAD

**Issue by VincentLanglet** [palantir/tslint#4531]
### Bug Report

- __TSLint version__: 5.12.0
- __TypeScript version__: 3.3.3
- __Running TSLint via__:  CLI / PhpStorm

#### TypeScript code being linted

```ts
someFunction(param => param);
```

with `tslint.json` configuration:

```json
"typedef": [
      true,
      "call-signature",
      "arrow-call-signature",
      "parameter",
      "arrow-parameter",
      "property-declaration",
      "member-variable-declaration"
    ],
```

#### Actual behavior
No error

#### Expected behavior
Error, should ask for something like
```ts
someFunction((param: any): any => param);
```

**Comment by VincentLanglet** [comment]
Seems like a duplicate of https://github.com/palantir/tslint/issues/2000
